test/integration/aaa_download_only_test.go

```diff
@@ -275,7 +274,7 @@ func TestBinaryMirror(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), Minutes(10))
 	defer Cleanup(t, profile, cancel)
 
-	tmpDir, err := ioutil.TempDir("", "kb_test")
+	tmpDir, err := os.MkdirTemp("", "kb_test")
```

Comment:
```suggestion
	tmpDir := t.TempDir()
```

https://pkg.go.dev/testing#T.TempDir

Reply:
I didn't know this was a thing, thanks for informing me!